scripts/download_data_file

```diff
@@ -120,7 +120,7 @@ function dev_file::edit() {
       dev_string="${file_name},${dev_string}"
     done
 
-  echo "REACT_APP_FORCE_STATIC=${dev_string}" >> "${TRACKERDIR}/frontend/.env.development"
+  echo "REACT_APP_FORCE_STATIC=geographies.json,${dev_string}" >> "${TRACKERDIR}/frontend/.env.development"
```

Comment:
it's always in the /tmp but it needs to be in the env file to look for it. Try removing it from your env.development and restart your server, on mine the map cards don't load

Reply:
Yeah I know but the way we generate the dev string is that we just read every file in the directory right?

Reply:
oh i see what you mean. hm i wonder why mine crashed then. i mustve lost it when i was copy pasting strings around because i was having lots of issues with the 100mb limit